lib/experimental/script-modules.php

```diff
@@ -200,3 +200,29 @@ function gutenberg_dequeue_module( $module_identifier ) {
 	_deprecated_function( __FUNCTION__, 'Gutenberg 17.6.0', 'wp_dequeue_script_module' );
 	wp_script_modules()->dequeue( $module_identifier );
 }
+
+/**
+ * Registers Gutenberg Script Modules.
+ *
+ * @since 19.3
+ */
+function gutenberg_register_script_modules() {
```

Comment:
This is the simplest possible version of this function. Right now script module registration is scattered across several places in blocks and interactivity API.

I think all (re-)registration should happen here, but that's a refactor that can happen separately from this PR.